recipes/liboping/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

To generate liboping-config.cmake